### ISSUE TITLE
> Is there a chance that the page double rendering bug could happen without strict mode? We are seeing double api calls with React 18 but do not seem to have strict mode enabled.

### ISSUE THREAD

**Issue by sennymunoz** [facebook/react#26229]
> Is there a chance that the page double rendering bug could happen without strict mode? We are seeing double api calls with React 18 but do not seem to have strict mode enabled.

I had a similar issue with disabled Strictmode. In my case the rerendering happened, because in a specific function, i was not using `event.preventDefault()`. Adding this line stopped the double rendering.

_Originally posted by @StGrSt in https://github.com/facebook/react/issues/24467#issuecomment-1443442930_

**Comment by tony1658** [comment]
I don't think this is a bug, neither related to React. See: https://stackoverflow.com/a/50819392 

If you truly think there is a bug, please show a reproducible example.
